### ISSUE TITLE
Microsoft.Graph.Applications Version Number

### ISSUE THREAD

**Issue by andrew-s-taylor** [cisagov/ScubaGear#463]
# 🐛 Summary #

Required Versions is looking for up to 1.99 of Microsoft.Graph.Applications, but latest version is 2.2

## To reproduce ##

1. Install-Module microsoft.graph.authentications
2. Run Scuba

## Expected behavior ##

What did you expect to happen that didn't?

Failed to complete, module won't import

## Any helpful log output or screenshots ##

Paste the results here:

```console
Import-Module : No acceptable installed version found for module: Microsoft.Graph.Applications
        Required Min Version: 1.14.0 | Max Version: 1.99.99999
        Run Get-InstalledModule to see a list of currently installed modules
        Run SetUp.ps1 or Install-Module Microsoft.Graph.Applications -Force -MaximumVersion 1.99.99999 to install the 
latest acceptable version of Microsoft.Graph.Applications


```

Add any screenshots of the problem here.

**Comment by gdasher** [comment]
Thank you for the feedback.  We will look at updating the tool to support version 2 of this module in a future release.  We need to determine backwards compatibility first, but hopefully it will be a minor lift and we can get it into our next release along with the baseline updates.

You should be able to install a 1.x version of the module explicitly using Install-Module -MaximumVersion 1.99.9999

**Comment by gdasher** [comment]
I took a look.  Obstacles to upgrading to version 2 of Graph Powershell:

1. Change to behavior of "beta" cmdlets (no more Get-MgProfile/Select-MgProfile).  Now the beta cmdlets are in a different set of "beta" packages.  Its easy to handle this by catching/ignoring the failure to select "beta" BUT
1. We appear to use "beta" cmdlets in at least one place (Get-MgDirectorySetting in AAD 2.7).  This is now Get-MgBetaDirectorySetting.  We will need to identify a way to handle this, either by setting minimum version of ScubaGear to Graph 2.x and changing how we call it or by handling both cases to preserve backwards compatibility.  This is likely a topic for once the baseline updates are finalized but before the code is updated.

**Comment by gdasher** [comment]
This issue is also responsible for all the PowerShell and Smoke test failures.

**Comment by crutchfield** [comment]
Closed with #514
